src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceObject = {};
```

Comment:
The variable name 'sourceObject' is not very descriptive. Consider renaming it to something more indicative of its purpose, such as 'styleObject' or 'cssProperties'.